manifests/k/Kitware/CMake/3.22.5/Kitware.CMake.locale.en-US.yaml

```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Kitware.CMake
+PackageVersion: 3.22.5
+PackageLocale: en-US
+Publisher: Kitware
+PublisherUrl: https://www.kitware.com/
+PublisherSupportUrl: https://www.kitware.com/contact-us/
+PrivacyUrl: https://www.kitware.com/privacy/#privacy-policy
+Author: Kitware
+PackageName: CMake
+PackageUrl: https://cmake.org/
+License: Copyright 2000-2022 Kitware, Inc. and Contributors
```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate